src/backend/database/task.model.js

```diff
@@ -0,0 +1,10 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("./connection.js");
+
+const Task = sequelize.define("Task", {
+  id: { type: DataTypes.STRING, allowNull: false, primaryKey: true },
```

Comment:
change the `id` type to `UUID` v4. [see this example](https://sequelize.org/docs/v7/models/data-types/#built-in-default-values-for-uuid) for more detail

Reply:
and, don't forget to format your document to get more readable

Reply:
mb, done sir